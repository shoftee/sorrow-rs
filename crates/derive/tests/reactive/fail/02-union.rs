use sorrow_derive::Reactive;

#[derive(Reactive)]
#[repr(C)]
pub union TestUnion {
    field_one: i32,
    field_two: i32,
}

fn main() {}
