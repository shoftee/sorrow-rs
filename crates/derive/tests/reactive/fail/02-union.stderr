error: derive(Reactive) is only supported on struct types.
 --> tests/reactive/fail/02-union.rs:5:5
  |
5 | pub union TestUnion {
  |     ^^^^^
